prow/plugins/approve/approvers/README.md

```diff
@@ -129,13 +132,15 @@ K8s-bot updates comment:
 	You can indicate your approval by writing `/approve` in a comment
 	You can cancel your approval by writing `/approve cancel` in a comment
 
-**Case (b)**: *approver3* is NOT an approver of the PR. S/he writes `/approve`
+### Step 3:
+
+*approver3* is NOT an approver of the PR. S/he writes `/approve`
 
 K8s-bot updates comment:
 	
 	[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
-	This pull-request has been approved by:* approver3, PRAuthor* 
+	This pull-request has been approved by: *approver1, approver3, PRAuthor* 
 	We suggest the following additional approvers: **approver1,** **approver2**
```

Comment:
We wouldn't suggest `approver1` in this case since they already approved.

Reply:
Yes, thanks!